virtcontainers/fc.go

```diff
@@ -525,7 +525,6 @@ func (fc *firecracker) fcAddNetDevice(endpoint Endpoint) error {
 		GuestMac:          endpoint.HardwareAddr(),
 		IfaceID:           &ifaceID,
 		HostDevName:       endpoint.NetworkPair().TapInterface.TAPIface.Name,
-		State:             "Attached",
```

Comment:
I guess you need to update FC vendor for it as well?

Reply:
@bergwolf Makes sense, even though that API change was already present before and I missed it as we were using an older firecracker binary. Revendored to match 0.12.0.